lib/src/credentials.dart

```diff
@@ -13,6 +13,8 @@ import 'handle_access_token_response.dart';
 import 'parameters.dart';
 import 'utils.dart';
 
+typedef OnCredentialsRefreshed = void Function(Credentials);
```

Comment:
This needs to be documented, just explain what and why you would implement this...

Reply:
Is this a good type name?

Perhaps we should call it `CredentialsRefreshedCallback`, is the `On-` pattern used in other code?

Reply:
Right, just noticed that the `XXXCallback` naming is used in the Dart library.